EXILED/Exiled.API/Features/Toys/Light.cs

```diff
@@ -153,7 +153,7 @@ public static Light Create(Vector3? position /*= null*/, Vector3? rotation /*= n
         /// <returns>The corresponding <see cref="LightSourceToy"/> instance.</returns>
         public static Light Get(LightSourceToy lightSourceToy)
```

Comment:
You know what we need to break this Get Method and doing it similar to Piclup API for example (that more a suggestion than i could work on if you want

Reply:
I can do it, but in different PR. I'm planing to completely rework Admin Toys section in API =)